### DISCUSSION TITLE
QSV setup in Docker using Portainer

### DISCUSSION THREAD

**Discussion by Vendo232** [immich-app/immich#4658] [Q&A] [answered]
the hwaccel guide mentiones this:

Setup
If you do not already have it, download the latest [hwaccel.yml](https://github.com/immich-app/immich/releases/latest/download/hwaccel.yml) file and ensure it's in the same folder as the docker-compose.yml.

if I use Portainer/Stack deployment where should I position this file?

**Comment by mertalev** [comment, marked as answer]
You can inline the fields in this file. For QSV, just copy over the `/dev/dri` line into `immich-microservices`.

**Comment by Vendo232** [comment]
thank you sir, added this to the stack

immich-microservices:
    container_name: immich_microservices
    image: ghcr.io/immich-app/immich-server:release
    entrypoint: ["/bin/sh", "./start-microservices.sh"]
    devices:
      - /dev/dri:/dev/dri
    volumes:
      - /mnt/media/pictures:/usr/src/app/upload
    env_file:
      - stack.env
      -
